src/components/Carousal.jsx

```diff
@@ -0,0 +1,61 @@
+import React, {Component, useState} from 'react'
+import {Carousel} from 'react-bootstrap'
+import Bivakzones from '../bivakzones.json'
+
+
+export default (props)=>{
+    
+    const bivakzone = props.bivakzone;
+    
+    const [index, setIndex] = useState(0);
+     
+  
+    const handleSelect = (selectedIndex, e) => {
+      setIndex(selectedIndex);
+    };
+
+    let imgsURL=[];
+    let imgsKeyArr=[];
+    let image="image";
+ 
+    if (bivakzone){
+        console.log(bivakzone)
+       
+        let propertiesKeys= Object.keys(bivakzone.properties);
+        console.log(propertiesKeys)
+       

```

Comment:
These blank spaces and also the linting stages of var's and functions should be neatly edited before making a PR